docs/utils/data.rst

```diff
@@ -0,0 +1,220 @@
+with a few helpers provided by `~astropy.utils.data`.
+
+If your cache starts behaving oddly you can use
+`~astropy.utils.data.check_download_cache` to examine your cache contents and
+raise an exception if it finds any anomalies.  If a single file is undesired or
+damaged, it can be removed by calling
+`~astropy.utils.data.clear_download_cache` with an argument that is the URL it
+was obtained from, the filename of the downloaded file, or the hash of its
+contents. Should the cache ever become badly corrupted,
+`~astropy.utils.data.clear_download_cache` with no arguments will simply delete
+the whole directory, freeing the space and removing any inconsistent data. Of
+course, if you remove data using either of these tools, any processes currently
+using that data may be disrupted (or, under Windows, deleting the cache may not
+be possible until those processes terminate). So use
+`~astropy.utils.data.clear_download_cache` with care.
+
+To check the total space occupied by the cache, use
+`~astropy.utils.data.cache_total_size`. The contents of the cache can be
+listed with `~astropy.utils.data.get_cached_urls`, and the presence of a
+particular URL in the cache can be tested with
+`~astropy.utils.data.is_url_in_cache`. More general manipulations can be
+carried out using `~astropy.utils.data.cache_contents`, which returns a
+dict mapping URLs to on-disk filenames of their contents.
+
+If you want to transfer the cache to another computer, or preserve its contents
+for later use, you can use the functions `~astropy.utils.data.export_download_cache` to
+produce a zipfile listing some or all of the cache contents, and
+`~astropy.utils.data.import_download_cache` to load the astropy cache from such a
+zipfile.
+
+Using Astropy With Limited or No Internet Access
+================================================
+
+You might want to use astropy on a telescope control machine behind a strict
+firewall. Or you might be running contionuous integration on your Astropy
+server and want to avoid hammering astronomy servers on every pull request for
+every architecture. Or you might not have access to US government or military
+web servers. Whichever is the case, you may need to avoid Astropy needing data
+from the Internet. There is no simple and complete solution to this problem at
+the moment, but there are tools that can help.
+
+Exactly which external data your project depends on will depend on what parts
+of Astropy you use and how. The most general solution is to use a computer that
+can access the Internet to run a version of your calculation that pulls in all
+the data files you will require, including sufficiently up-to-date versions of
+files like the IERS data that update regularly. Then once the cache on this
+connected machine is loaded with everything necessary, transport the cache
+contents to your target machine by whatever means you have available, whether
+by copying via an intermediate machine, sneakernet, or carrier pigeon. The
```

Comment:
I don't want to start triaging bugs caused by carrier pigeon (biological or otherwise), so let's reword this. 😅 
```suggestion
by copying via an intermediate machine or portable disk drives. The
```

Reply:
https://tools.ietf.org/html/rfc2549